lib/components/GGanttGrid.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <div
+    ref="g-grid-container"
+    class="g-grid-container"
+    :style="{ left: `${rowLabelWidth}px`, width: `${timeCount * gridSize}px` }"
+  >
+    <div
+      v-for="(childPoint, index) in allChildPoints"
+      :key="index"
+      :class="[
+        'g-grid-line',
+        { 'g-grid-line-last': index === allChildPoints.length - 1 },
+        {
+          'g-grid-line-highlighted':
+            (precision === 'day' && highlightedHours.includes(childPoint)) ||
+            (precision === 'month' && highlightedDays.includes(childPoint))
+        }
+      ]"
+      :style="{ width: `${gridSize}px` }"
+    />
+  </div>
+</template>
+
+<script>
+import moment from 'moment'
+
+export default {
+  name: 'GGanttGrid',
+
+  props: {
+    chartStart: { type: String },
+    chartEnd: { type: String },
+    rowLabelWidth: { type: Number },
+    highlightedHours: { type: Array, default: () => [] },
+    highlightedDays: { type: Array, default: () => [] },
+    precision: { type: String },
+    timeCount: { type: Number },
+    gridSize: { type: Number }
+  },
+
+  computed: {
+    allChildPoints() {
+      let start = moment(this.chartStart)
+      let end = moment(this.chartEnd)
+      let res = []
```

Comment:
```suggestion
      const start = moment(this.chartStart)
      const end = moment(this.chartEnd)
      const res = []
```
At no time is the reference modified, therefore it does not have to be a LET.